playbooks/files/export-dcn.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Script to execute on the central site to generate useful
+# data to deploy edge sites.
+
+unset OS_CLOUD
+export OS_AUTH_TYPE=none
+export OS_ENDPOINT=http://127.0.0.1:8006/v1/admin
+DIR=/home/stack/exported-data
+
+mkdir -p $DIR
+openstack stack output show standalone EndpointMap --format json \
+| jq '{"parameter_defaults": {"EndpointMapOverride": .output_value}}' \
```

Comment:
Don't you need `--raw-output` here?

Reply:
I didn't need it for the script to run fine, I can add it if you want, although i didn't test with this arg.